internal/tba/tba.go

```diff
@@ -301,8 +301,38 @@ func (s *Service) GetTeamKeys(ctx context.Context, eventKey string) ([]string, e
 	return teamKeys, err
 }
 
+// GetTeams retrieves all teams
+func (s *Service) GetTeams(ctx context.Context) ([]store.Team, error) {
+	allTeams := []store.Team{}
+	for page := 0; ; page++ {
```

Comment:
Can we add a constant that sets a reasonable limit for this? Like in case their API just kept responding with teams forever.